substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/configure/ConfigurationFiles.java

```diff
@@ -91,6 +91,9 @@
         @Option(help = "Resources describing predefined classes that can be loaded at runtime.", type = OptionType.User)//
         public static final HostedOptionKey<LocatableMultiOptionValue.Strings> PredefinedClassesConfigurationResources = new HostedOptionKey<>(new LocatableMultiOptionValue.Strings());
 
+        @Option(help = "Causes unknown attributes in configuration objects to cause an exception instead of emitting a warning.")//
+        public static final HostedOptionKey<Boolean> StrictConfig = new HostedOptionKey<>(false);
```

Comment:
I think this should be called `StrictConfiguration` since configuration is spelled out in the other option names too, and the description should say that it aborts the build rather than causing an exception, which is more user-centric.

Reply:
I agree completely, I've changed the option name and the description